modules/core/src/test/config/log4j-test.xml

```diff
@@ -141,6 +141,10 @@
         <level value="INFO"/>
     </category>
 
+    <category name="org.apache.calcite">
```

Comment:
What is this category used for? The previous category does the same.

Reply:
removed